### PR TITLE
Add teams dropdown for employee

### PR DESCRIPTION
**Pull request by vladPinteaFrontmen**
# Changes
Added dropdown to set team to an employee on employee manage view

## Related issues
https://github.com/FrontMen/fm-hours/issues/60

## Added
Added dropdown to set team to an employee on employee manage view

## How to test
1. Go to /employees
2. Go to manage on an employee
3. Change value in team dropdown and save
4. Refresh to see value retrieved or check inside firestore

## Screenshots
image
firestore dev example
image
image
rule in firestore
image

### CODE REVIEW

**Comment by YejunWu168** [comment]
Currently an employee without team shows nothing in select. I'd suggest to add a default value for no team assigned

**Comment by lassecapel** [comment]
Team names: Classics, New Fronteers, Upfront, FrontX, Eindhoven, NonDutch

File: pages/employees/_employee_id.vue [hunk 1]
```diff
@@ -176,16 +192,23 @@ export default defineComponent({
         store.dispatch("customers/getCustomers");
       }
 
-      if (store.getters["employees/adminList"].length === 0) {
-        store.dispatch("employees/getAdminList");
-      }
+      store.dispatch("employees/getAdminList");
```
Comment (mmehmetAliIzci) [inline pages/employees/_employee_id.vue:195]: Is there reason why we ditched getting admin list only if there is none ?

Comment (vladPinteaFrontmen) [reply pages/employees/_employee_id.vue:196]: it's already handled inside the action, so it was redundant

**Comment by vladPinteaFrontmen** [comment]
@YejunWu168 updated

**Review by YejunWu168 — APPROVED** [review]

**Pull request merged**